components/gitpod-cli/cmd/timeout-extend.go

```diff
@@ -39,7 +39,7 @@ var extendTimeoutCmd = &cobra.Command{
 			}
 			fail(err.Error())
 		}
-		fmt.Println("Workspace timeout has been extended to three hours.")
+		fmt.Println("Workspace timeout is extended to 180 minutes.")
```

Comment:
I do also think we should consider deprecating, or updating the behaviour of this command: 

- https://github.com/gitpod-io/gitpod/issues/16083

Rather than updating it.